src/EnergyPlus/CommandLineInterface.cc

```diff
@@ -792,7 +792,7 @@ namespace CommandLineInterface {
         bool NewHeading = false;
 
         while (inputFile.good() && !Found) {
-            const auto readResult = inputFile.readLine();
+            const auto &readResult = inputFile.readLine();
```

Comment:
Same comment as above. Have I mentioned how much I hate `auto`? This is why I don't know what to say about these because I don't know what types they are. This is obnoxious.